src/async-components/views/dialogs/security/CreateSecretStorageDialog.tsx

```diff
@@ -739,7 +740,18 @@ export default class CreateSecretStorageDialog extends React.PureComponent<IProp
                             onClick={this.onCopyClick}
                             disabled={this.state.phase === Phase.Storing}
                         >
-                            { this.state.copied ? _t("Copied!") : _t("Copy") }
+                            <span
+                                className="mx_CreateSecretStorageDialog_recoveryKeyCopyButtonText"
+                                style={{ height: this.state.copied ? '0' : 'auto' }}
+                            >
+                                { _t("Copy") }
+                            </span>
+                            <span
+                                className="mx_CreateSecretStorageDialog_recoveryKeyCopyButtonText"
+                                style={{ height: this.state.copied ? 'auto' : '0' }}
+                            >
+                                { _t("Copied!") }
+                            </span>
```

Comment:
It would be important to add `aria-hidden` here. Otherwise a screenreader would say it as "Copy Copied!".

```suggestion
                            <span
                                className="mx_CreateSecretStorageDialog_recoveryKeyCopyButtonText"
                                style={{ height: this.state.copied ? '0' : 'auto' }}
                                aria-hidden={this.state.copied}
                            >
                                { _t("Copy") }
                            </span>
                            <span
                                className="mx_CreateSecretStorageDialog_recoveryKeyCopyButtonText"
                                style={{ height: this.state.copied ? 'auto' : '0' }}
                                aria-hidden={!this.state.copied}
                            >
                                { _t("Copied!") }
                            </span>
```